packages/retail-ui-extensions/src/components/TextField/TextField.ts

```diff
@@ -1,31 +1,42 @@
 import {createRemoteComponent} from '@remote-ui/core';
-import type {AutoCapitalizationType} from '../shared';
 
-export type InputType =
-  | 'text'
-  | 'number'
-  | 'percent'
-  | 'boundlessPercent'
-  | 'currency'
-  | 'giftcard'
-  | 'expiry'
-  | 'date'
-  | 'zip'
-  | 'password'
-  | 'email'
-  | 'url'
-  | 'phone';
+interface ActionProps {
```

Comment:
I'm not sure but do I need to export these..?

Reply:
I think we should export these, in case partners want to do logic outside of the component's render. Make sure you add the export at all the index files